src/doc/rustc/src/codegen-options/index.md

```diff
@@ -172,6 +172,25 @@ values:
 
 The default if not specified depends on the target.
 
+## hint-mostly-unused
```

Comment:
Given that the flag is now unstable/nightly-gated, this documentation should be moved to the unstable-book (under `src/doc/unstable-book/src/compiler-flags`) and a tracking issue should be created.

Reply:
No need for a tracking issue before consensus tho right ? This still needs fcp anyways. (And given the reservations in the thread some more tests and all will be needed before stabilization.)

Reply:
Sure, the creation of the tracking issue can wait until the MCP is accepted and the PR nearing approval.

Reply:
I've updated this to be an unstable option and documented in the unstable book.